### PR TITLE
Comments Query Loop: Update default template

### PR DESCRIPTION
**Pull request by DAreRodz**
## What?

Changes the default template to make it more aesthetic. Also, it modifies the default values of some block attributes.

## Why?

To let editor users have a first good structure for the Comments Query Loop inner blocks.

## How?

- Changing the initial `innerBlocks` for the Comments Query Loop block.
- Changing the default value of `isLink` and `fontSize` attributes of
   - Comment Author Name
   - Comment Date
   - Comment Edit Link
   - Comment Reply Link

## Testing Instructions

1. Open a post or page
2. Add a Comments Query Loop block
3. Check that the default block organization looks like in the image below.

## Screenshots or screencast 

Screenshot 2022-04-07 at 17 05 35

### CODE REVIEW

**Review by SantosGuillamot — APPROVED** [review]
Regarding the user experience, it looks better to me 🙂 Thanks for the PR David!

File: packages/block-library/src/comment-author-name/index.php [hunk 1]
```diff
@@ -27,6 +27,9 @@ function render_block_core_comment_author_name( $attributes, $content, $block )
 	if ( isset( $attributes['textAlign'] ) ) {
 		$classes .= 'has-text-align-' . esc_attr( $attributes['textAlign'] );
 	}
+	if ( isset( $attributes['fontSize'] ) ) {
+		$classes .= 'has-' . esc_attr( $attributes['fontSize'] ) . '-font-size';
```
Comment (michalczaplinski) [inline packages/block-library/src/comment-author-name/index.php:31]: I think there is no need to escape the attributes here, they are [already escaped in `get_block_wrapper_attributes()`](https://github.com/WordPress/wordpress-develop/blob/0ea7a38664746f9ad4f402768975c66928817531/src/wp-includes/class-wp-block-supports.php#L210).

File: packages/block-library/src/comment-date/index.php [hunk 2]
```diff
@@ -23,7 +23,12 @@ function render_block_core_comment_date( $attributes, $content, $block ) {
 		return '';
 	}
 
-	$wrapper_attributes = get_block_wrapper_attributes();
+	$classes = '';
+	if ( isset( $attributes['fontSize'] ) ) {
+		$classes .= 'has-' . esc_attr( $attributes['fontSize'] ) . '-font-size';
```
Comment (michalczaplinski) [inline packages/block-library/src/comment-date/index.php:28]: I think there is no need to escape the attributes here, they are [already escaped in `get_block_wrapper_attributes()`](https://github.com/WordPress/wordpress-develop/blob/0ea7a38664746f9ad4f402768975c66928817531/src/wp-includes/class-wp-block-supports.php#L210).

File: packages/block-library/src/comment-edit-link/index.php [hunk 3]
```diff
@@ -31,6 +31,9 @@ function render_block_core_comment_edit_link( $attributes, $content, $block ) {
 	if ( isset( $attributes['textAlign'] ) ) {
 		$classes .= 'has-text-align-' . esc_attr( $attributes['textAlign'] );
 	}
+	if ( isset( $attributes['fontSize'] ) ) {
+		$classes .= 'has-' . esc_attr( $attributes['fontSize'] ) . '-font-size';
```
Comment (michalczaplinski) [inline packages/block-library/src/comment-edit-link/index.php:35]: I think there is no need to escape the attributes here, they are [already escaped in `get_block_wrapper_attributes()`](https://github.com/WordPress/wordpress-develop/blob/0ea7a38664746f9ad4f402768975c66928817531/src/wp-includes/class-wp-block-supports.php#L210).

File: packages/block-library/src/comment-reply-link/index.php [hunk 4]
```diff
@@ -53,7 +53,10 @@ function render_block_core_comment_reply_link( $attributes, $content, $block ) {
 
 	$classes = '';
 	if ( isset( $attributes['textAlign'] ) ) {
-		$classes .= 'has-text-align-' . $attributes['textAlign'];
+		$classes .= 'has-text-align-' . esc_attr( $attributes['textAlign'] );
+	}
+	if ( isset( $attributes['fontSize'] ) ) {
+		$classes .= 'has-' . esc_attr( $attributes['fontSize'] ) . '-font-size';
```
Comment (michalczaplinski) [inline packages/block-library/src/comment-reply-link/index.php:59]: I think there is no need to escape the attributes here, they are [already escaped in `get_block_wrapper_attributes()`](https://github.com/WordPress/wordpress-develop/blob/0ea7a38664746f9ad4f402768975c66928817531/src/wp-includes/class-wp-block-supports.php#L210).

**Review by michalczaplinski — APPROVED** [review]
Looks great! 👍 

I'm happy to approve it already as the suggestions are non-blocking and can be done in a follow-up PR.

(the video got cut-off in a weird way, sorry about that)

https://user-images.githubusercontent.com/5417266/162333152-884118bb-7fc6-46f3-99f7-a37a5bd67795.mp4

**Comment by jasmussen** [comment]
Nice, this is a big improvement. Before:
before

After:

after

A couple of thoughts that can probably all be followed up on:

**Left column:** I wish we could use something other than the Columns block. As created, this becomes a bit rigid; in the GIF above, I try increasing the size of the avatar and it overflows the column. There's a technical question here: should the "Size" option become a "Width" option instead, and have access to unit control? Then, a la how Images work, we could potentially let the _resolution_ of the avatar be available in a dropdown, rather than be inextricably tied to the dimensions of the avatar:

Screenshot 2022-04-08 at 08 04 55

Ultimately this would make it possible for us to set the default _width_ of the avatar block to be `100%`, which would then correspond to the width of the column container it was inside. So when you resized the column, it'd also resize the image.

**Right column:** Looks great. The use of the Row block is good. 

**Avatar radius:** Right now it's preset to 20px, which of course is half of 40px. That makes it size inevenly. A trick here: if the border-radius size is set to _pixels_, it will never cause an ovaloid shape. I.e. if you set the radius to `9999px`, the image would remain a perfect circle right up until it's width is 2000px or more. It's only when the radius is set to % that the shape becomes ovaloid. In other words, so long as we set a default px border radius that is much larger than we think the image will ever be used, it'll stay circular. I'd recommend bumping it to the max.

**Comment by jasmussen** [comment]
To clarify the columns comment: if the avatar block had a percentage width and scaled to its column, the columns block is arguably the right one to use. It's mainly that there are multiple widths you need to update in order to change the avatar size, that's a bit of a challenge at the moment.

File: packages/block-library/src/comment-author-name/index.php (hunk 1 above)
Comment (cbravobernal) [reply packages/block-library/src/comment-author-name/index.php:31]: In this case, we don't use the `get_block_wrapper_attributes()` function to display that class, I guess that's the reason why we have it there.

**Comment by DAreRodz** [comment]
Thanks, @michalczaplinski, and @jasmussen for your useful feedback. I'll work on the suggested changes in a follow-up PR. 😁

File: phpunit/class-block-library-comment-template-test.php [hunk 5]
```diff
@@ -160,7 +163,7 @@ function test_rendering_comment_template_nested() {
 
 		$this->assertEquals(
 			gutenberg_render_block_core_comment_template( null, null, $block ),
```
Comment (gziolo) [inline phpunit/class-block-library-comment-template-test.php:165]: The expected value should go first here as well.

File: packages/block-library/src/comment-date/index.php (hunk 2 above)
Comment (carolinan) [reply packages/block-library/src/comment-date/index.php:28]: Was there an issue created for this? It has not been solved.

Comment (DAreRodz) [reply packages/block-library/src/comment-date/index.php:28]: Hey @carolinan, you're right; there's no issue created. 😅 Feel free to open one and address it.

**Pull request merged**